src/common/k8s-api/kube-object.store.ts

```diff
@@ -192,9 +191,9 @@ export abstract class KubeObjectStore<T extends KubeObject> extends ItemStore<T>
 
       return items;
     } catch (error) {
-      if (error.message) {
-        Notifications.error(error.message);
-      }
+      // if (error.message) {
```

Comment:
@Nokel81 is it ok to remove this Notification? Mixing UI with lower level classes is not a good idea (and it does not work in `main`).

Reply:
I think it should be fine. My only concern is that because no downstream consumer of this function call will be able to observe this error so a consumer on the renderer wouldn't be able to show the notification anyway.

Reply:
Fair enough.